### PR TITLE
Added Book info for show page Closes #20

### LINKED ISSUE #20: User Story 10, Book Show Page
```
As a Visitor,
When I visit a book's show page,
I see the following book information:
- the book title
- the author(s) who wrote the book
- the number of pages in the book
- the year the book was published
- a large image of the book cover

I also see a list of reviews for that book.
Each review will have a title and user, a numeric rating
from 1 to 5, and text for the review itself, and all content
must be present for each review.
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written
- [ ] edge case code written
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #21

child of #33

### PR DESCRIPTION
**Pull request by djc00p**
Added all the book info and test are passing

### CODE REVIEW

File: app/views/books/show.html.erb [hunk 1]
```diff
@@ -1,4 +1,27 @@
-<h1>Top Three Reviews</h1>
+<h1><%= @book.title %></h1>
+<section>
```
Comment (earl-stephens) [inline app/views/books/show.html.erb]: Do we need this section?

File: app/views/books/show.html.erb [hunk 2]
```diff
@@ -1,4 +1,27 @@
...
+  <ul>
+    <li>Author(s): <%= @book.lists_authors(@book) %></li>
+    <li>Pages: <%= @book.pages %></li>
+    <li>Year Published: <%= @book.year_pub %></li>
+  </ul>
+</section>
+<section class="show_reviews">
+  <h2>Reviews</h2>
+  <% @book.reviews.each do |review| %>
+  <section id="review">
```
Comment (earl-stephens) [inline app/views/books/show.html.erb]: I'm wondering if we should make these id's more specific, such as book_show_reviews (book for book resources, show for show page and review for the section)?  I'm just thinking ahead to when we style and the possibility of having multiple CSS selectors across the whole site with the same name...
